milestone_project_1_original.py

```diff
@@ -0,0 +1,121 @@
+"""
+1. Display a board
+2. User can input 1-9
+3. Check if the user input is taken, else, ask for another number
+4. Add the mark to the board
+5. Check if the user wins or tie, else, next player's turn
+6. Play again?
+"""
+
+board = ['-',' ',' ',' ',' ',' ',' ',' ',' ',' ']
+current_player = "Player_1"
+player_1_mark, player_2_mark = ("O","X")
+
+def display_board(board):
+    print("Game Board      Position Guide")
+    print("  " + board[1] + "|" + board[2] + "|" + board[3] + "             1|2|3")
+    print("  -----             -----")
+    print("  " + board[4] + "|" + board[5] + "|" + board[6] + "             4|5|6")
+    print("  -----             -----")
+    print("  " + board[7] + "|" + board[8] + "|" + board[9] + "             7|8|9")
+
+def ask_player_1_input():
+    print("Player_1's turn, your mark is 'X'")
+    position = int(input("Please choose a number from 1-9: "))
+
+    while position not in range(1,10) or board[position] == "X" or board[position] == "O":
+        position = int(input("Invalid number or placement, please try again: "))
+
+    board[position] = "X"
+
+def ask_player_2_input():

```

Comment:
Most implementation are the same between these two functions, please use one function with parameter instead.

Reply:
fixed